v22.2/cost-based-optimizer.md

```diff
@@ -159,6 +157,47 @@ sql_stats_automatic_collection_min_stale_rows = 2000);
 
 Automatic statistics rules are checked once per minute. While altered automatic statistics table settings take immediate effect for any subsequent DML statements on a table, running row mutations that started prior to modifying the table settings may still trigger statistics collection based on the settings that existed before you ran the `ALTER TABLE ... SET` statement.
 
+### Enable and disable forecasted statistics for tables
+
+You can enable and disable [forecasted statistics](show-statistics.html#display-forecasted-statistics) collection for individual tables using the `sql_stats_forecasts_enabled` table parameter. This table setting **takes precedence** over the `sql.stats.forecasts.enabled` [cluster setting](cluster-settings.html).
+
+You can either configure this setting during table creation:
+
+~~~ sql
+CREATE TABLE accounts (
+    id INT PRIMARY KEY,
+    balance DECIMAL)
+WITH (sql_stats_forecasts_enabled = false);
+~~~
+
+Or by using [`ALTER TABLE ... SET`](set-storage-parameter.html):
+
+~~~ sql
+CREATE TABLE accounts (
+    id INT PRIMARY KEY,
+    balance DECIMAL);
+
+ALTER TABLE accounts
+SET (sql_stats_forecasts_enabled = false);
+~~~
+
+The current table settings are shown in the `WITH` clause output of `SHOW CREATE TABLE`:
+
+~~~ sql
```

Comment:
Nit: You can remove the sql syntax highlighting from the output block here. And, the columns are slightly off, small adjustment needed.

Reply:
(I also think there is a output block like this earlier on the page that you could take the opportunity to edit!) 😺